forgerock-auth/src/androidTest/java/org/forgerock/android/auth/RootDeviceTest.java

```diff
@@ -34,8 +34,9 @@ public void testDangerousPropertyDetector() {
         RootDetector rootDetector = FRRootDetector.builder()
                 .detector(new DangerousPropertyDetector())
                 .build();
+        int sdkVersion = android.os.Build.VERSION.SDK_INT;
```

Comment:
Can be `static` and `final` at class level once it's used on multiple tests?

Reply:
Updated